src/Symfony/Component/Messenger/Bridge/AmazonSqs/Transport/Connection.php

```diff
@@ -124,6 +127,9 @@ public static function fromDsn(string $dsn, array $options = [], HttpClientInter
             'accessKeyId' => urldecode($parsedUrl['user'] ?? '') ?: $options['access_key'] ?? self::DEFAULT_OPTIONS['access_key'],
             'accessKeySecret' => urldecode($parsedUrl['pass'] ?? '') ?: $options['secret_key'] ?? self::DEFAULT_OPTIONS['secret_key'],
         ];
+        if (isset($options['debug'])) {
+            $clientConfiguration['debug'] = $options['debug'];
```

Comment:
This requires `async-aws/core: ^1.5`. The bridge's `composer.json` should be updated